docs/content/docs/concepts/keypairs.mdx

```diff
@@ -264,37 +267,48 @@ The public key associated with a given private key can be used by anyone to veri
 If someone sends us a message and signature that they claim to be from the example above, we could use Bob's public key to verify that the signature was in fact the one produced by Bob and that the message was not modified in transit.
 
 ```ts twoslash
-import { SignatureBytes } from "@solana/kit";
+import { SignatureBytes } from '@solana/kit';
 const bobsPublicKey = null as unknown as CryptoKey;
 const supposedlyBobsSignature = null as unknown as SignatureBytes;
 // ---cut-before---
-import { getUtf8Encoder, verifySignature } from "@solana/kit";
+import { getUtf8Encoder, verifySignature } from '@solana/kit';
 const verificationSucceeded = await verifySignature(
-  bobsPublicKey,
-  supposedlyBobsSignature,
-  getUtf8Encoder().encode("The meeting is at 6:00pm"),
+    bobsPublicKey,
+    supposedlyBobsSignature,
+    getUtf8Encoder().encode('The meeting is at 6:00pm'),
 );
 if (!verificationSucceeded) {
-  throw new Error("Either the message was modified, the signature was not produced by Bob, or both");
+    throw new Error(
+        'Either the message was modified, the signature was not produced by Bob, or both',
+    );
 }
 ```
 
 Verification protects equally against false claims of who produced the signature as it does against false claims about what message was signed. Both of these will return `false`;
 
+{/* prettier-ignore */}
```

Comment:
This was brutal to figure out. All because of that `mallorysSignature` highlight below. This entire code block is now unformatted. You have to remove the annotation, format, then replace the annotation to make edits.